### ISSUE TITLE
Conflicting import, imported name 'NavArgs' is ambiguous`

### ISSUE THREAD

**Issue by jahongir9779** [raamcosta/compose-destinations#122]
Hi, thanx for the new feature, I've upgraded to the new version and got this compilation error. Any ideas how can I fix it? 

`...NavArgsGetters.kt: (5, 84): Conflicting import, imported name 'NavArgs' is ambiguous`

This is generated class
```
import androidx.lifecycle.SavedStateHandle
import uz.uzkassa.apay.coreui.composables.destinations.ErrorBottomSheetDestination
import uz.uzkassa.apay.coreui.composables.destinations.ErrorBottomSheetDestination.NavArgs
import uz.uzkassa.apay.coreui.composables.destinations.ErrorInfoBottomSheetDestination
import uz.uzkassa.apay.coreui.composables.destinations.ErrorInfoBottomSheetDestination.NavArgs
import uz.uzkassa.apay.coreui.composables.destinations.PayloadBottomSheetScreenDestination
import uz.uzkassa.apay.coreui.composables.destinations.PayloadBottomSheetScreenDestination.NavArgs

inline fun <reified T> SavedStateHandle.navArgs(): T {
    return navArgs(T::class.java, this)
}

@Suppress("UNCHECKED_CAST")
fun <T> navArgs(argsClass: Class<T>, savedStateHandle: SavedStateHandle): T {
    return when (argsClass) {
		ErrorBottomSheetDestination.NavArgs::class.java -> ErrorBottomSheetDestination.argsFrom(savedStateHandle) as T
		ErrorInfoBottomSheetDestination.NavArgs::class.java -> ErrorInfoBottomSheetDestination.argsFrom(savedStateHandle) as T
		PayloadBottomSheetScreenDestination.NavArgs::class.java -> PayloadBottomSheetScreenDestination.argsFrom(savedStateHandle) as T
        else -> error("Impossible due to this method being updated automatically on build")
    }
}
```

**Comment by raamcosta** [comment]
Uh 🤦 

This is a bug with newest version, I’ll fix it today and release a new version. In the mean time you can use the version before the latest.
sorry for the inconvenience!

**Comment by raamcosta** [comment]
This is fixed and 1.5.4-beta is gonna be ready in the next hour or so from maven central.

Thank you for reporting @jahongir9779!

**Comment by raamcosta** [comment]
It was a very obvious and easy one ;)
